### ISSUE TITLE
1.7.10 crash

### ISSUE THREAD

**Issue by Honon** [TinyModularThings/IC2Classic#254]
Iron Scaffold. Occurred with right and left click without having anything.
```
---- Minecraft Crash Report ----
// This doesn't make any sense!

Time: 6/10/18 10:00 PM
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at net.minecraft.item.ItemStack.func_77969_a(ItemStack.java:356)
	at ic2.core.block.BlockScaffold.func_149737_a(BlockScaffold.java:76)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_78743_b(PlayerControllerMP.java:177)
	at net.minecraft.client.Minecraft.func_147116_af(Minecraft.java:1409)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1948)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.minecraft.item.ItemStack.func_77969_a(ItemStack.java:356)
	at ic2.core.block.BlockScaffold.func_149737_a(BlockScaffold.java:76)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_78743_b(PlayerControllerMP.java:177)
	at net.minecraft.client.Minecraft.func_147116_af(Minecraft.java:1409)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1948)
```

**Comment by Honon** [comment]
Version 1.2.1.6
Forge 10.13.4.1614

**Comment by Speiger** [comment]
@Honon modlist is also required since this is basically a impossible crash. (Unless someone asm something away)

**Comment by Honon** [comment]
I'm sorry. Is this OK?
If it is not enough, add it.
```
---- Minecraft Crash Report ----
// This doesn't make any sense!

Time: 6/10/18 10:00 PM
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at net.minecraft.item.ItemStack.func_77969_a(ItemStack.java:356)
	at ic2.core.block.BlockScaffold.func_149737_a(BlockScaffold.java:76)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_78743_b(PlayerControllerMP.java:177)
	at net.minecraft.client.Minecraft.func_147116_af(Minecraft.java:1409)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1948)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.minecraft.item.ItemStack.func_77969_a(ItemStack.java:356)
	at ic2.core.block.BlockScaffold.func_149737_a(BlockScaffold.java:76)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_78743_b(PlayerControllerMP.java:177)
	at net.minecraft.client.Minecraft.func_147116_af(Minecraft.java:1409)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1948)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityClientPlayerMP['shigre200'/98968, l='MpServer', x=53.02, y=65.62, z=-192.70]]
	Chunk stats: MultiplayerChunkCache: 289, 289
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (-96,65,-149), Chunk: (at 0,4,11 in -6,-10; contains blocks -96,0,-160 to -81,255,-145), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Level time: 1813890 game time, 1813890 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: 33 total; [EntityBabySkeleton['ベビースケルトン'/98436, l='MpServer', x=104.50, y=48.00, z=-244.50], EntitySpider['クモ'/98439, l='MpServer', x=105.56, y=48.00, z=-246.50], EntitySlime['スライム'/71, l='MpServer', x=0.31, y=28.14, z=-207.69], EntitySpider['クモ'/98377, l='MpServer', x=83.59, y=19.00, z=-222.81], EntitySheep['ヒツジ'/72, l='MpServer', x=36.97, y=68.00, z=-255.94], EntityZombie['ゾンビ'/98761, l='MpServer', x=75.25, y=22.00, z=-254.50], EntityZombie['ゾンビ'/99081, l='MpServer', x=74.50, y=53.00, z=-246.50], EntityZombie['ゾンビ'/98760, l='MpServer', x=77.50, y=22.00, z=-253.50], EntityCreeper['クリーパー'/99080, l='MpServer', x=108.84, y=64.00, z=-176.84], EntityClientPlayerMP['shigre200'/98968, l='MpServer', x=53.02, y=65.62, z=-192.70], EntityZombie['ゾンビ'/99083, l='MpServer', x=74.50, y=53.00, z=-247.50], EntityZombie['ゾンビ'/99082, l='MpServer', x=72.50, y=53.00, z=-249.50], EntityBabySkeleton['ベビースケルトン'/98639, l='MpServer', x=76.50, y=19.00, z=-222.50], EntityChicken['ニワトリ'/79, l='MpServer', x=-5.44, y=68.00, z=-248.41], EntityChicken['ニワトリ'/80, l='MpServer', x=109.53, y=62.39, z=-229.34], EntityCow['ウシ'/83, l='MpServer', x=100.63, y=63.00, z=-242.78], EntitySkeleton['スケルトン'/98644, l='MpServer', x=81.50, y=19.00, z=-221.50], EntityBabySkeleton['ベビースケルトン'/98014, l='MpServer', x=101.50, y=40.00, z=-183.50], EntityBabySkeleton['ベビースケルトン'/99107, l='MpServer', x=102.88, y=64.00, z=-179.88], EntityBabySkeleton['ベビースケルトン'/99106, l='MpServer', x=99.38, y=64.00, z=-176.38], EntitySkeleton['スケルトン'/99111, l='MpServer', x=101.53, y=64.00, z=-183.09], EntitySkeleton['スケルトン'/98343, l='MpServer', x=48.09, y=14.00, z=-245.50], EntitySkeleton['スケルトン'/99110, l='MpServer', x=105.50, y=64.00, z=-178.50], EntityZombie['ゾンビ'/99115, l='MpServer', x=94.28, y=64.00, z=-179.59], EntityZombie['ゾンビ'/99116, l='MpServer', x=105.34, y=64.00, z=-173.66], EntityMechanicalArm['不明'/111, l='MpServer', x=-47.00, y=64.00, z=-110.00], EntityBlock['不明'/112, l='MpServer', x=-46.25, y=68.25, z=-65.38], EntityBlock['不明'/113, l='MpServer', x=-29.75, y=27.00, z=-65.38], EntityBlock['不明'/114, l='MpServer', x=-29.75, y=68.25, z=-108.25], EntityBlock['不明'/115, l='MpServer', x=-29.60, y=26.00, z=-65.23], EntityZombie['ゾンビ'/99061, l='MpServer', x=76.50, y=20.00, z=-220.50], EntitySkeleton['スケルトン'/98997, l='MpServer', x=6.50, y=70.00, z=-226.84], EntityCreeper['クリーパー'/98680, l='MpServer', x=73.66, y=20.00, z=-207.00]]
	Retry entities: 0 total; []
	Server brand: thermos,cauldron,craftbukkit,mcpc,kcauldron,fml,forge
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2444)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:927)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 751066416 bytes (716 MB) / 2366111744 bytes (2256 MB) up to 3817865216 bytes (3641 MB)
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4096m -Xms256m -XX:PermSize=256m
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 15, tallocated: 95
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 55 mods loaded, 55 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJA	Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJA	CodeChickenCore{1.0.7.47} [CodeChicken Core] (minecraft.jar) 
	UCHIJA	intelliinput{2.6} [IntelliInput for 1.7.10] (minecraft.jar) 
	UCHIJA	NotEnoughItems{1.0.5.120} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.5.120-universal.jar) 
	UCHIJA	<CoFH ASM>{000} [CoFH ASM] (minecraft.jar) 
	UCHIJA	FastCraft{1.25} [FastCraft] (fastcraft-1.25.jar) 
	UCHIJA	CoFHCore{1.7.10R3.1.4} [CoFH Core] (CoFHCore-[1.7.10]3.1.4-329.jar) 
	UCHIJA	BuildCraft|Core{7.1.23} [BuildCraft] (buildcraft-7.1.23.jar) 
	UCHIJA	BuildCraft|Transport{7.1.23} [BC Transport] (buildcraft-7.1.23.jar) 
	UCHIJA	BuildCraft|Factory{7.1.23} [BC Factory] (buildcraft-7.1.23.jar) 
	UCHIJA	BuildCraft|Silicon{7.1.23} [BC Silicon] (buildcraft-7.1.23.jar) 
	UCHIJA	BuildCraft|Robotics{7.1.23} [BC Robotics] (buildcraft-7.1.23.jar) 
	UCHIJA	BuildCraft|Energy{7.1.23} [BC Energy] (buildcraft-7.1.23.jar) 
	UCHIJA	BuildCraft|Builders{7.1.23} [BC Builders] (buildcraft-7.1.23.jar) 
	UCHIJA	CarpentersBlocks{3.3.8.1} [Carpenter's Blocks] (Carpenter's Blocks v3.3.8.1 - MC 1.7.10.jar) 
	UCHIJA	MineTweaker3{3.0.13} [MineTweaker 3] (CraftTweaker-1.7.10-3.0.13.jar) 
	UCHIJA	PTRModelLib{1.0.0} [PTRModelLib] (Decocraft-2.4.2_1.7.10.jar) 
	UCHIJA	props{2.4.2} [Decocraft] (Decocraft-2.4.2_1.7.10.jar) 
	UCHIJA	fastleafdecay{1.4} [Fast Leaf Decay] (FastLeafDecay-1.7.10-1.4.jar) 
	UCHIJA	IC2{2.2.999} [Industrial Craft Classic] (IC2Classic Version 1.2.1.6.jar) 
	UCHIJA	IC2-Classic-Spmod{0.0.0.0} [IC2 Classic Detection Helper] (IC2Classic Version 1.2.1.6.jar) 
	UCHIJA	ThermalFoundation{1.7.10R1.2.6} [Thermal Foundation] (ThermalFoundation-[1.7.10]1.2.6-118.jar) 
	UCHIJA	ImmersiveEngineering{0.7.7} [Immersive Engineering] (ImmersiveEngineering-0.7.7.jar) 
	UCHIJA	LunatriusCore{1.1.2.21} [LunatriusCore] (LunatriusCore-1.7.10-1.1.2.21-universal.jar) 
	UCHIJA	InGameInfoXML{2.8.1.82} [InGame Info XML] (InGameInfoXML-1.7.10-2.8.1.82-universal.jar) 
	UCHIJA	inpure|core{1.7.10R1.0.0B9} [INpureCore] (INpureCore-[1.7.10]1.0.0B9-62.jar) 
	UCHIJA	inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks] (InventoryTweaks-1.59-dev-152.jar) 
	UCHIJA	IronChest{6.0.62.742} [Iron Chest] (ironchest-1.7.10-6.0.62.742-universal.jar) 
	UCHIJA	irontank{@VERSION@ build @BUILD@} [Iron Tanks] (irontanks-1.1.16.jar) 
	UCHIJA	journeymap{5.1.4p2} [JourneyMap] (journeymap-1.7.10-5.1.4p2-unlimited.jar) 
	UCHIJA	ThermalExpansion{1.7.10R4.1.5} [Thermal Expansion] (ThermalExpansion-[1.7.10]4.1.5-248.jar) 
	UCHIJA	LogisticsPipes{0.9.3.132} [Logistics Pipes] (logisticspipes-0.9.3.132.jar) 
	UCHIJA	ForgeMultipart{1.1.2.331} [Forge Multipart] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) 
	UCHIJA	Mekanism{9.1.1} [Mekanism] (Mekanism-1.7.10-9.1.1.1031.jar) 
	UCHIJA	MekanismGenerators{9.1.1} [MekanismGenerators] (MekanismGenerators-1.7.10-9.1.1.1031.jar) 
	UCHIJA	modtweaker2{0.9.6} [Mod Tweaker 2] (ModTweaker2-0.9.6.jar) 
	UCHIJA	Morpheus{1.7.10-1.6.21} [Morpheus] (Morpheus-1.7.10-1.6.21.jar) 
	UCHIJA	NEIAddons{1.12.14.40} [NEI Addons] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCHIJA	NEIAddons|Developer{1.12.14.40} [NEI Addons: Developer Tools] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCHIJA	NEIAddons|AppEng{1.12.14.40} [NEI Addons: Applied Energistics 2] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCHIJA	NEIAddons|Botany{1.12.14.40} [NEI Addons: Botany] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCHIJA	NEIAddons|Forestry{1.12.14.40} [NEI Addons: Forestry] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCHIJA	NEIAddons|CraftingTables{1.12.14.40} [NEI Addons: Crafting Tables] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCHIJA	NEIAddons|ExNihilo{1.12.14.40} [NEI Addons: Ex Nihilo] (neiaddons-1.12.14.40-mc1.7.10.jar) 
	UCHIJA	neiintegration{1.1.2} [NEI Integration] (NEIIntegration-MC1.7.10-1.1.2.jar) 
	UCHIJA	netherportalfix{1.0} [Nether Portal Fix] (netherportalfix-mc1.7.10-1.1.0.jar) 
	UCHIJA	StorageDrawers{1.7.10-1.10.9} [Storage Drawers] (StorageDrawers-1.7.10-1.10.9.jar) 
	UCHIJA	ThermalDynamics{1.7.10R1.2.1} [Thermal Dynamics] (ThermalDynamics-[1.7.10]1.2.1-172.jar) 
	UCHIJA	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10.jar) 
	UCHIJA	WailaHarvestability{1.1.6} [Waila Harvestability] (WailaHarvestability-mc1.7.10-1.1.6.jar) 
	UCHIJA	wawla{1.3.1} [What Are We Looking At] (Wawla-1.0.5.120.jar) 
	UCHIJA	McMultipart{1.1.2.331} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) 
	UCHIJA	ForgeMicroblock{1.1.2.331} [Forge Microblocks] (ForgeMultipart-1.7.10-1.1.2.331-universal.jar) 
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 388.13' Renderer: 'GeForce GTX 1060 6GB/PCIe/SSE2'
	CoFHCore: -[1.7.10]3.1.4-329
	ThermalFoundation: -[1.7.10]1.2.6-118
	ThermalExpansion: -[1.7.10]4.1.5-248
	ThermalDynamics: -[1.7.10]1.2.1-172
	Launched Version: forge-10.13.4.1614
	LWJGL: 2.9.1
	OpenGL: GeForce GTX 1060 6GB/PCIe/SSE2 GL version 4.6.0 NVIDIA 388.13, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: 日本語 (日本)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: On (16)
```

**Comment by Speiger** [comment]
I know what it is as soon as i update mods again this gets a patch.
